installation/index.md

```diff
@@ -7,9 +7,8 @@ title: Installation Overview
 
 # Installation Overview
 
-openHAB 2 is based on the Eclipse SmartHome framework and is fully written in Java.
-As such, it only depends on a Java Virtual Machine, which is available for many platforms.
-openHAB can be executed on different versions of **macOS** and **Windows** and many different variants of **Linux** (Ubuntu, Raspbian, ...).
+OpenHAB 2 is based on the Eclipse SmartHome framework which is fully written in Java. As such, OpenHAB requires a Java Virtual Machine to be installed on the system you intend to install openHAB on. A Java Vitual Machine is available for many platforms.
```

Comment:
Please change this part to one line per sentence.
The website build mechanism will take care of the design and we can track changes better, with only one sentence for each line.